PostHog/PostHogSDK.swift

```diff
@@ -510,8 +520,9 @@ let maxRetryDelay = 30.0
         return false
     }
 
-    @objc(captureWithEvent:properties:userProperties:userPropertiesSetOnce:groups:)
+    @objc(captureWithEvent:distinctId:properties:userProperties:userPropertiesSetOnce:groups:)
     public func capture(_ event: String,
+                        distinctId distinctIdParam: String? = nil,
```

Comment:
Can we just name `distinctId: String? = nil,` instead? why do we need `distinctIdParam`?